pkg/datapath/linux/config/config.go

```diff
@@ -245,6 +245,9 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 		}
 		if option.Config.EnableHostPort {
 			cDefinesMap["ENABLE_HOSTPORT"] = "1"
+			if option.Config.HostPortOnlyLocal {
+				cDefinesMap["HOSTPORT_ONLY_LOCAL"] = "1"
```

Comment:
I can't find the relevant macro in the datapath (or anywhere else). Is there some BPF code that you forgot to add to the commit? Or am I missing something?

Reply:
Agree, the only use for this statement seems to be offline introspection that this feature is enabled. I think that we should be able to determine this without the header define.

Reply:
I just want to say it is relevant to the "Fixes: 7fcf660",  and that is ok to remove it.

About the HOSTPORT_ONLY_LOCAL macro, it is really useless until now.
Actually I intended to use this macro to optimize the datapath in future, for example: skip the iptable rules and redirect the traffics from bpf_netdev to bpf_lxc directly for the hostport-only endpoints.  (because I found an issue that the traffics come from outside to access the hostport service, but the request direction traffics still be routed to host iptable conntracking because of the NO_REDIRECT macro)
I will remove this macro in this fix.